### ISSUE TITLE
Error updating from local files: Unexpected character '29' found when expecting key.

### ISSUE THREAD

**Issue by SebPlaysGamesYT** [Theo47/depressurizer#120]
I get this error when I start the program, and once it's open my library is incomplete.
![depressurizer error](https://user-images.githubusercontent.com/19425166/28283431-a13758be-6b25-11e7-91fb-e9e47cf2b004.PNG)

**Comment by mvegter** [comment]
@SebPlaysGamesYT Does this still happen? Have you tried reinstalling Depressurizer?

**Comment by SebPlaysGamesYT** [comment]
Yes, this still happens. I just tried reinstalling Depressurizer, I still have the same issue.

**Comment by Theo47** [comment]
Can you go to Steam/appcache and upload appinfo.vdf and packageinfo.vdf?

Also try deleting those 2 files, restart steam and then Depressurizer and check if the error persists

p.s: You are using latest version (v0.7.3.0), right?
